README.md

```diff
@@ -67,7 +67,7 @@ https://nodejs.org/calendar
 
 <sup>Click `+GoogleCalendar` at the bottom right to add to your own Google calendar.</sup>
 
-All calendar maintainers have `Make changes AND manage sharing` permissions. If you would like to help maintain your team's calendar events, open a PR adding your name to the list above. Once approved, one of the calendar maintainers will add you to the calendar settings.
+All calendar maintainers have `Make changes AND manage sharing` permissions. If you would like to help maintain your team's calendar events, open a PR adding your name to the list below. Once approved, one of the calendar maintainers will add you to the calendar settings.
```

Comment:
I'm fairly sure this should've been "below" :)

Reply:
+1